ui/app/i18n/registration/locale_pt_BR.json

```diff
@@ -152,6 +152,11 @@
     "REGISTRATION_OPTION_NONE": "Nenhum",
     "REGISTRATION_OPTION_NONE_TEXT": "Documento",
     "BMI STATUS": "Estado Nutricional",
-    "BMI": "IMC"
+    "BMI": "IMC",
+    "REGISTRATION_TRANSFER_OUT_INFO_KEY": "Transferido Para",
+    "REGISTRATION_TRANSFER_OUT_HF_NAME_INFO":"Unidade Sanitária de Destino",
+    "Is_Transferred_Out":"Data de Transferência",
```

Comment:
Should be: `Is Transferred Out`

Reply:
Updated translation "Is_Transferred_Out":"Transferir Para",